### ISSUE TITLE
Installation problem under 1.3.4 and 1.3.5

### ISSUE THREAD

**Issue by nsfw** [marcusb/grails-cometd#3]
Basically it looks like the dependencies on cometd-java-server aren't getting downloaded... or some such!

Happy to provide any additional information! But unfortunately I'm not sure what's most useful for you.

FYI - in case something about my installation was inconsistent I've done a full uninstall of grails (including nuking ~/.grails and ~/.ivy2), but am getting the same results below. 

--snip-- 

org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
/Users/scott/.grails/1.3.5/projects/concierge/plugins/cometd-0.2.1/CometdGrailsPlugin.groovy: 21: unable to resolve class org.cometd.server.BayeuxServerImpl
 @ line 21, column 1.
   import org.cometd.server.BayeuxServerImpl
   ^

/Users/scott/.grails/1.3.5/projects/concierge/plugins/cometd-0.2.1/CometdGrailsPlugin.groovy: 23: unable to resolve class org.cometd.bayeux.server.BayeuxServer
 @ line 23, column 1.
   import org.cometd.bayeux.server.BayeuxServer
   ^

/Users/scott/.grails/1.3.5/projects/concierge/plugins/cometd-0.2.1/CometdGrailsPlugin.groovy: 22: unable to resolve class org.cometd.server.CometdServlet
 @ line 22, column 1.
   import org.cometd.server.CometdServlet
   ^

--snip--
3 errors

**Comment by nsfw** [comment]
Here's a more complete listing of the build process prior to the above:

Resolving dependencies...
Downloading: http://repo1.maven.org/maven2/org/cometd/java/cometd-java-client/2.0.beta1/cometd-java-client-2.0.beta1.pom ...
Download complete.
Downloading: http://repo1.maven.org/maven2/org/cometd/java/cometd-java-client/2.0.beta1/cometd-java-client-2.0.beta1.pom.sha1 ...
Download complete.
Downloading: http://repo1.maven.org/maven2/org/eclipse/jetty/jetty-io/7.1.3.v20100526/jetty-io-7.1.3.v20100526.pom ...
Download complete.
Downloading: http://repo1.maven.org/maven2/org/eclipse/jetty/jetty-io/7.1.3.v20100526/jetty-io-7.1.3.v20100526.pom.sha1 ...
Download complete.
Downloading: http://repo1.maven.org/maven2/org/eclipse/jetty/jetty-http/7.1.3.v20100526/jetty-http-7.1.3.v20100526.pom ...
Download complete.
Downloading: http://repo1.maven.org/maven2/org/eclipse/jetty/jetty-http/7.1.3.v20100526/jetty-http-7.1.3.v20100526.pom.sha1 ...
Download complete.
Downloading: http://repo1.maven.org/maven2/org/eclipse/jetty/jetty-client/7.1.3.v20100526/jetty-client-7.1.3.v20100526.pom ...
Download complete.
Downloading: http://repo1.maven.org/maven2/org/eclipse/jetty/jetty-client/7.1.3.v20100526/jetty-client-7.1.3.v20100526.pom.sha1 ...
Download complete.
Downloading: http://repo1.maven.org/maven2/org/cometd/java/cometd-java-client/2.0.beta1/cometd-java-client-2.0.beta1.jar ...
Download complete.
Downloading: http://repo1.maven.org/maven2/org/cometd/java/cometd-java-client/2.0.beta1/cometd-java-client-2.0.beta1.jar.sha1 ...
Download complete.
Downloading: http://repo1.maven.org/maven2/org/eclipse/jetty/jetty-io/7.1.3.v20100526/jetty-io-7.1.3.v20100526.jar ...
Download complete.
Downloading: http://repo1.maven.org/maven2/org/eclipse/jetty/jetty-io/7.1.3.v20100526/jetty-io-7.1.3.v20100526.jar.sha1 ...
Download complete.
Downloading: http://repo1.maven.org/maven2/org/eclipse/jetty/jetty-http/7.1.3.v20100526/jetty-http-7.1.3.v20100526.jar ...
Download complete.
Downloading: http://repo1.maven.org/maven2/org/eclipse/jetty/jetty-http/7.1.3.v20100526/jetty-http-7.1.3.v20100526.jar.sha1 ...
Download complete.
Downloading: http://repo1.maven.org/maven2/org/eclipse/jetty/jetty-client/7.1.3.v20100526/jetty-client-7.1.3.v20100526.jar ...
Download complete.
Downloading: http://repo1.maven.org/maven2/org/eclipse/jetty/jetty-client/7.1.3.v20100526/jetty-client-7.1.3.v20100526.jar.sha1 ...
Download complete.
Dependencies resolved in 24853ms.
Running script /Users/scott/grails/scripts/RunApp.groovy
Environment set to development
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
/Users/scott/.grails/1.3.5/projects/concierge/plugins/cometd-0.2.1/CometdGrailsPlugin.groovy: 21: unable to resolve class org.cometd.server.BayeuxServerImpl
 @ line 21, column 1.
   import org.cometd.server.BayeuxServerImpl
   ^

/Users/scott/.grails/1.3.5/projects/concierge/plugins/cometd-0.2.1/CometdGrailsPlugin.groovy: 23: unable to resolve class org.cometd.bayeux.server.BayeuxServer
 @ line 23, column 1.
   import org.cometd.bayeux.server.BayeuxServer
   ^

/Users/scott/.grails/1.3.5/projects/concierge/plugins/cometd-0.2.1/CometdGrailsPlugin.groovy: 22: unable to resolve class org.cometd.server.CometdServlet
 @ line 22, column 1.
   import org.cometd.server.CometdServlet
   ^

3 errors

```
at org.codehaus.groovy.control.ErrorCollector.failIfErrors(ErrorCollector.java:296)
at org.codehaus.groovy.control.CompilationUnit.applyToSourceUnits(CompilationUnit.java:829)
at org.codehaus.groovy.control.CompilationUnit.doPhaseOperation(CompilationUnit.java:511)
at org.codehaus.groovy.control.CompilationUnit.processPhaseOperations(CompilationUnit.java:487)
at org.codehaus.groovy.control.CompilationUnit.compile(CompilationUnit.java:464)
at org.codehaus.groovy.control.CompilationUnit.compile(CompilationUnit.java:448)
at _PluginDependencies_groovy$_run_closure5_closure38.doCall(_PluginDependencies_groovy:246)
at _PluginDependencies_groovy$_run_closure5_closure38.doCall(_PluginDependencies_groovy)
at _GrailsSettings_groovy$_run_closure10.doCall(_GrailsSettings_groovy:280)
at _GrailsSettings_groovy$_run_closure10.call(_GrailsSettings_groovy)
at _PluginDependencies_groovy$_run_closure5.doCall(_PluginDependencies_groovy:245)
at _GrailsPackage_groovy$_run_closure2.doCall(_GrailsPackage_groovy:167)
at RunApp$_run_closure1.doCall(RunApp.groovy:28)
at gant.Gant$_dispatch_closure5.doCall(Gant.groovy:381)
at gant.Gant$_dispatch_closure7.doCall(Gant.groovy:415)
at gant.Gant$_dispatch_closure7.doCall(Gant.groovy)
at gant.Gant.withBuildListeners(Gant.groovy:427)
at gant.Gant.this$2$withBuildListeners(Gant.groovy)
at gant.Gant$this$2$withBuildListeners.callCurrent(Unknown Source)
at gant.Gant.dispatch(Gant.groovy:415)
at gant.Gant.this$2$dispatch(Gant.groovy)
at gant.Gant.invokeMethod(Gant.groovy)
at gant.Gant.executeTargets(Gant.groovy:590)
at gant.Gant.executeTargets(Gant.groovy:589)
```

Error loading plugin manager: startup failed:
/Users/scott/.grails/1.3.5/projects/concierge/plugins/cometd-0.2.1/CometdGrailsPlugin.groovy: 21: unable to resolve class org.cometd.server.BayeuxServerImpl
 @ line 21, column 1.
   import org.cometd.server.BayeuxServerImpl
   ^

/Users/scott/.grails/1.3.5/projects/concierge/plugins/cometd-0.2.1/CometdGrailsPlugin.groovy: 23: unable to resolve class org.cometd.bayeux.server.BayeuxServer
 @ line 23, column 1.
   import org.cometd.bayeux.server.BayeuxServer
   ^

/Users/scott/.grails/1.3.5/projects/concierge/plugins/cometd-0.2.1/CometdGrailsPlugin.groovy: 22: unable to resolve class org.cometd.server.CometdServlet
 @ line 22, column 1.
   import org.cometd.server.CometdServlet
   ^

3 errors

**Comment by marcusb** [comment]
I haven't seen any such issues, and I'm using it with Grails 1.3.4 and 1.3.5. It looks more like a Grails problem.Please try "grails compile".

**Comment by nsfw** [comment]
OK - found an issue, and a fix. 

When rebuilding the stand alone plugin (i.e. grails run-app in the plugin directory) I got "bad version # in class" error. I never saw this error when including the plugin in my app... but I was using Java 1.5, so I switched to 1.6 and it built with no errrors, as did my application.

So, what I think is happening is the maven repo with cometd-server-java, etc... was compiled with 1.6

Why the error message didn't show up that way when using it as a plugin, I'm not sure exactly. But I'm pretty sure that was the issue.

Thanks for writing the plugin!
